src/main/resources/avro/metadata.avdl

```diff
@@ -114,4 +135,42 @@ record Dataset {
 
 }
 
+/**
+An analysis contains an interpretation of one or several experiments.
+(e.g. SNVs, copy number variations, methylation status) together with
+information about the methodology used.
+*/
+record Analysis {
+  /** The analysis UUID. This is globally unique. */
```

Comment:
This is formulated differently in the metadata branch. You should pick up the version from there:

https://github.com/ga4gh/schemas/blob/metadata/src/main/resources/avro/metadata.avdl#L767

Also, IDs are described in https://github.com/ga4gh/schemas/blob/metadata/doc/source/metadata_intro.rst. Not sure if this the right place in the long run...